oai_agents/agents/agent_utils.py

```diff
@@ -65,32 +66,55 @@ def __init__(self, obs_space):
         self.observation_space = obs_space
 
 class CustomAgent():
-    def __init__(self, name, args, start_position, action):
+    def __init__(self, args, name, trajectories):
+        self.args = args
         self.name = f'CA_{name}'
-        self.action = Action.ACTION_TO_INDEX[action]
         self.policy = CustomPolicy(spaces.Dict({'visual_obs': spaces.Box(0,1,(1,))}))
```

Comment:
For a readability, we might want to capsulate spaces.Dict({'visual_obs': spaces.Box(0,1,(1,))}) into a function. By just reading it, we don't know what we're trying to pass.

Reply:
It does not matter what self.policy is, it only exists to avoid getting errors. I can put a comment next to it for explanation